docs/csharp/programming-guide/strings/index.md

```diff
@@ -53,7 +53,7 @@ For more information about how to create new strings that are based on modificat
 
 Beginning with C# 11, you can use *raw string literals* to more easily create strings that are multi-line, or use any characters requiring escape sequences. *Raw string literals* remove the need to ever use escape sequences. You can write the string, including whitespace formatting, how you want it to appear in output. A *raw string literal*:
 
-- Starts and ends with a sequence of at least three double quote characters (`"""`). You're allowed more than three consecutive characters to start and end the sequence in order to support string literals that contain three (or more) repeated quote characters.
+- Starts and ends with a sequence of at least three double quote characters (`"""`). You can use more than three consecutive characters to start and end the sequence. You can embed string literals that contain three (or more) repeated quote characters.
```

Comment:
I really feel like the previous explanation was clearer. Effectively these statements are independent now:

- You can use more than three consecutive characters to start and end the sequence.
- You can embed string literals that contain three (or more) repeated quote characters.

While the second statement _is_ true, it's only true (without the 'or more' clause) if you use more than three characters for the sequence. But that's not clear anymore.

Reply:
Right. I sometimes hate acrolinx. Let's revert:

```suggestion
- Starts and ends with a sequence of at least three double quote characters (`"""`). You can use more than three consecutive characters to start and end the sequence to support string literals that contain three (or more) repeated quote characters.
```